app/controllers/annotations_controller.rb

```diff
@@ -36,6 +51,14 @@ def update
   action_auth_level :destroy, :course_assistant
   def destroy
     @annotation.destroy
+
```

Comment:
Annotation's already been destroyed so it doesn't exist any more and doesn't need to be valid?